rhel/SPECS/pmm-managed.spec

```diff
@@ -45,7 +47,13 @@ ln -s $(pwd) src/%{provider_prefix}
 
 %build
 export GOPATH=$(pwd)
-go build -ldflags "${LDFLAGS:-} -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \n')" -a -v -x %{provider_prefix}
+
+export PMM_RELEASE_VERSION=%{full_pmm_version}
+export PMM_RELEASE_TIMESTAMP=$(date '+%s')
+export PMM_RELEASE_FULLCOMMIT=%{commit}
```

Comment:
`commit` is hardcoded above and wrong.

Reply:
> `commit` is hardcoded above and wrong.

no, in spec file it is just template, we get and change the commit during the build